config.py

```diff
@@ -12,35 +12,47 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Flask config."""
-import os
-from dotenv import load_dotenv
-
-load_dotenv()
+import secrets
 
+"""Flask config."""
 
 class Config:
     """Base config."""
 
     FLASK_ENV = 'production'
     TESTING = False
     DEBUG = False
-    
-    PHILIPS_HUE_URL = os.environ.get('PHILIPS_URL')
-    
-    
+
+
+
 class ProdConfig(Config):
-    pass
+
+    PHILIPS_HUE_IP = secrets.GoogleSecretManagerSecret(
+        'alertmanager-2020-intern-r', 'philips_ip')
+    PHILIPS_HUE_USERNAME = secrets.GoogleSecretManagerSecret(
+        'alertmanager-2020-intern-r', 'philips_username')
+
+    def __init__(self):
+        self.PHILIPS_HUE_IP = self.PHILIPS_HUE_IP.get_secret_value()
+        self.PHILIPS_HUE_USERNAME = self.PHILIPS_HUE_USERNAME.get_secret_value()
+
 
 
 class DevConfig(Config):
     FLASK_ENV = 'development'
     DEBUG = True
     TESTING = True
-    
+    PHILIPS_HUE_IP = secrets.EnvironmentVariableSecret('PHILIPS_HUE_IP')
+    PHILIPS_HUE_USERNAME = secrets.EnvironmentVariableSecret('PHILIPS_HUE_USERNAME')
```

Comment:
Done. I decided to continue using properties as opposed to regular attributes since this was how it was done in Flask's documentation regarding config files: https://flask.palletsprojects.com/en/1.1.x/config/